src/javascript/app/pages/trade/purchase.js

```diff
@@ -123,9 +123,9 @@ const Purchase = (() => {
                     dataManager.setPurchase({
                         error: {
                             ...error,
-                            signupUrl: signup_url,
-                            title    : localize('Ready to trade?'),
-                            isCustom : true,
+                            signup_url,
+                            title   : localize('Ready to trade?'),
+                            isCustom: true,
```

Comment:
```suggestion
                            is_custom: true,
```
i seems like renaming should be also done inside data-manager file for all non-function properties